message TVideoItem {
    required string Id = 1;
    optional string Name = 2;
    optional double Rating = 3;
    optional uint32 MinAge = 4;
};

message TOldRow {
    required string Key = 1;
    optional string Value = 2;
};

message TNewRow {
    required string Key = 1;
    optional string Value = 2;
    optional int32 Count = 3;
};

message TNewRowSmall {
    required string Key = 1;
};

message TNewRowStrict {
    required string Key = 1;
    optional string Value = 2;
    required int32 Count = 3;
};
